node/src/main/scala/org/bitcoins/node/util/BitcoinSpvNodeUtil.scala

```diff
@@ -38,9 +38,9 @@ trait BitcoinSpvNodeUtil extends BitcoinSLogger {
               loop(newRemainingBytes, message :: accum)
             }
           case Failure(exception) =>
-            logger.debug(
-              "Failed to parse network message, could be because tcp frame isn't aligned")
-            logger.debug(exception.getMessage)
+            logger.error(
```

Comment:
Figured it was best to up the log level, as the fact that this was on debug was the reason I didn't see it while working on the SPV node

Reply:
it's fairly common that TCP frames aren't aligned, so this may be noisy